tgui/packages/tgui/interfaces/PanDEMIC.tsx

```diff
@@ -0,0 +1,369 @@
+import { InfernoNode } from 'inferno';
+import { useBackend } from '../backend';
+import {
+  Button,
+  LabeledList,
+  Section,
+  Stack,
+  Table,
+  Tabs,
+} from '../components';
+import { Window } from '../layouts';
+
+interface PathogenSymptom {
+  name: string;
+  stealth: number;
+  resistance: number;
+  stageSpeed: number;
+  transmissibility: number;
+}
+
+interface PathogenStrain {
+  commonName?: string;
+  description?: string;
+  bloodDNA?: string;
+  bloodType?: string;
+  diseaseAgent: string;
+  possibleTreatments?: string;
+  transmissionRoute?: string;
+  symptoms?: PathogenSymptom[];
+  isAdvanced: 0 | 1;
+}
+
+interface PanDEMICData {
+  synthesisCooldown: 0 | 1;
+  beakerLoaded: 0 | 1;
+  beakerContainsBlood: 0 | 1;
+  beakerContainsVirus: 0 | 1;
```

Comment:
```suggestion
  synthesisCooldown: BooleanLike;
  beakerLoaded: BooleanLike;
  beakerContainsBlood: BooleanLike;
  beakerContainsVirus: BooleanLike;
```
You can also use just 'boolean'

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/69762909/3264b187-d00c-4831-af63-c36f86a0b8e2)